### ISSUE TITLE
images should have alt text

### ISSUE THREAD

**Issue by zkoppert** [github/github-ospo#18]
[Guide on writing effective alt text](https://webaim.org/techniques/alttext/)
- [ ] Add alt text to images in markdown files
- [ ] Remove the markdownlint exception for alt images in the file `.markdownlint-cli2.jsonc`
